source/containers.tex

```diff
@@ -19698,6 +19698,25 @@
 
 \rSec5[mdspan.layout.stride.cons]{Constructors}
 
+\indexlibraryctor{layout_stride::mapping}%
+\begin{itemdecl}
+constexpr mapping() noexcept;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\expects
+\tcode{layout_right::mapping<extents_type>().required_span_size()}
+is representable as a value of type \tcode{index_type}\iref{basic.fundamental}.
+
+\pnum
+\effects
+Direct-non-list-initializes \exposid{extents_} with \tcode{extents_type()}, and
+for all $d$ in the range  $[0, \exposid{rank_})$,
```

Comment:
```suggestion
for all $d$ in the range $[0, \exposid{rank_})$,
```
Please, remind me, why not `\range`?

Reply:
One could argule whether that's a code-zero or a maths-zero, but I don't think it matters much. I'll use the macro.

Reply:
That's unfortunately inconsistent. Sibling constructors, and probably other layout_*::mappings as well, still use the manual style.